iothub/device/samples/readme.md

```diff
@@ -10,16 +10,13 @@ This folder contains simple samples showing how to use the various features of M
 - [Receive message sample][d-receive-message-sample]
 - [Twin sample][d-twin-sample]
 - [File upload sample][d-file-upload-sample]
-- [Import/export devices sample][d-import-export-devices-sample]
```

Comment:
What happened to import/export? It didn't belong in the devices samples section, yeah?

Reply:
It was removed from the devices folder and moved to the services folder. https://github.com/Azure-Samples/azure-iot-samples-csharp/pull/319